src/Vatras/Data/Prop.agda

```diff
@@ -0,0 +1,34 @@
+module Vatras.Data.Prop where
+
+open import Data.Bool as Bool using (Bool; if_then_else_)
+
+data Prop (F : Set) : Set where
+  true  : Prop F
+  false : Prop F
+  var   : F → Prop F
+  ¬_    : Prop F → Prop F
+  _∧_   : Prop F → Prop F → Prop F
+
+infix 24 var
```

Comment:
What's your rational on this? Why not treat `var` as a normal function? I already tried removing this `infix 24 var` and nothing breaks.

Reply:
I do not remember. Removed it.